pkg/pheanstalk/PheanstalkConsumer.php

```diff
@@ -88,6 +88,13 @@ public function acknowledge(Message $message): void
      */
     public function reject(Message $message, bool $requeue = false): void
     {
+        InvalidMessageException::assertMessageInstanceOf($message, PheanstalkMessage::class);
+
+        if (false == $message->getJob()) {
+            $verb = $requeue ? 'requeued' : 'rejected';
+            throw new \LogicException("The message could not be $verb because it does not have job set.");
```

Comment:
```suggestion
            throw new \LogicException(sprintf(
                "The message could not be %s because it does not have job set.", 
                $requeue ? 'requeued' : 'rejected'
            ));
```